web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -148,10 +148,44 @@ function joinup_core_post_update_0106503(): void {
   }
 }
 
+/**
+ * Fix the creation date for the rdf graphs.
+ */
+function joinup_core_post_update_0106504(): void {
+  $query = <<<QUERY
+WITH <http://joinup.eu/bundle/rdf-graph/graph>
+INSERT { ?entity <http://purl.org/dc/terms/issued> ?creation_time }
+WHERE { 
+  ?entity <http://purl.org/dc/terms/modified> ?creation_time .
+  FILTER NOT EXISTS { ?entity <http://purl.org/dc/terms/issued> ?time }
+}
+QUERY;
+
+  \Drupal::getContainer()->get('sparql.endpoint')->query($query);
+}
+
+/**
+ * Clean up orphaned triples.
+ */
+function joinup_core_post_update_0106505(): void {
+  $query = <<<QUERY
+DELETE { GRAPH ?g { ?s ?p ?o } }
+WHERE {
+  GRAPH ?g {
+    ?s ?p ?o .
+    FILTER NOT EXISTS {?s <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> ?type} .
+    VALUES ?g { <http://joinup.eu/asset_distribution/published> <http://joinup.eu/asset_release/published> <http://joinup.eu/asset_release/draft> <http://joinup.eu/collection/published> <http://joinup.eu/collection/draft> <http://joinup.eu/contact-information/published> <http://joinup.eu/licence/published> <http://joinup.eu/owner/published> <http://joinup.eu/provenance_activity> <http://joinup.eu/solution/published> <http://joinup.eu/solution/draft><http://joinup.eu/spdx_licence/published> }
+  }
+}
+QUERY;
+
+  \Drupal::getContainer()->get('sparql.endpoint')->query($query);
+}
+
 /**
  * Clean up the validation graphs.
  */
-function joinup_core_post_update_0106504(array &$sandbox): void {
+function joinup_core_post_update_0106506(array &$sandbox): void {
```

Comment:
Looks a little bit odd to add the new (post)updates before the ones already committed in develop except the case when one can influence the other. Is this the case?

I noticed this because I've run the all the updates against `develop` and dump the DB in `./tmp/mysql.sql` just o avoid running all path alias updates every time. Because of the changing of the order in this PR, suddenly I saw some updates, such as _Clean up orphaned triples_ not running. The reason was that the `0106505` already ran with `develop`. For this reason I think it's better, as much as we can, to preserve the order that is coming naturally from the development flow.